src/services/users/update-refresh-token.js

```diff
@@ -7,6 +7,7 @@ const updateRefreshToken = async(refreshToken , id) => {
    WHERE id = $2 `,
 		[refreshToken , id]
 	)
+	return db.query
```

Comment:
this is not what I meant @beblicarl , I suggested you add return to the line that has `db.query`